packages/job-worker/src/playout/model/PlayoutPartInstanceModel.ts

```diff
@@ -176,6 +176,11 @@ export interface PlayoutPartInstanceModel {
 	 * @param time Reported stopped time
 	 */
 	setReportedStoppedPlayback(time: Time): boolean
+	/**
+	 * Set the Reported stopped playback time, including still-playing PieceInstances
+	 * @param time Reported stopped time on all available objects
+	 */
+	setReportedStoppedPlaybackWithPieceInstances(time: Time): boolean
```

Comment:
should this be a public method on the model, or is this an internal implementation detail?

Reply:
This is a very good question! I didn't want to leak the implementation detail of how `pieceInstances` are organized and managed to the outside, because it didn't feel like whoever is calling this _should_ need to know what else needs to be called for the desired effect.

Would you say it makes more sense to just reuse `setReportedStoppedPlayback` and then have the caller update the individual pieceInstances using their model methods?

Reply:
I don't mind the method existing, my only thought was that this is called only by `PlayoutModelImpl` which knows that it must be working with a `PlayoutPartInstanceModelImpl`.
I now realise that the types don't quite line up with this, because of `this.currentPartInstance` returning the interface, but something could be done so that where you are calling `setReportedStoppedPlaybackWithPieceInstances` it knows it is the Impl type. 

I don't really mind either way though, might be best to leave it as it avoids needing some type fiddling